### ISSUE TITLE
ibmi_download_fix_status issue

### ISSUE THREAD

**Issue by cmh716** [IBM/ansible-for-i#154]
Does ibmi_download_fix_status work?

I successfully ordered and downloaded a PTF via Ansible, but this module does not work for me (the status of the order remains unknown).  I see that the module is doing this:

    sql_query_message_info = "SELECT MESSAGE_TIMESTAMP, MESSAGE_SECOND_LEVEL_TEXT FROM QSYS2.MESSAGE_QUEUE_INFO" \
        " WHERE MESSAGE_QUEUE_LIBRARY='QSYS' AND MESSAGE_QUEUE_NAME='QSERVICE' AND MESSAGE_ID='CPZ8C15' AND" \
        " MESSAGE_TYPE='COMPLETION' AND MESSAGE_SECOND_LEVEL_TEXT LIKE '%{order_id}%'"

The problem on my side is that the order_id does not exist in the message.  For example, the second level text for my order is this:

<img width="1090" alt="image" src="https://github.com/IBM/ansible-for-i/assets/3275478/7f28bdca-84c4-45ee-9097-4dea1811207d">

The Group PTFs I ordered were SF99667 and SF99665.  Our Primary language is 2924.

**Comment by cmh716** [comment]
I could probably query QUSRSYS/QASXPROB fields WSXPPTFID, WSXPPTFST and WSXPID, but of course the proper ansible module would be better.

**Comment by robgjertsen1** [comment]
I agree that this looks problematic if the second level text does not have an order id. I'll have to investigate further.
